test/Microsoft.AspNet.Authentication.Test/OAuthBearer/OAuthBearerMiddlewareTests.cs

```diff
@@ -27,6 +26,8 @@ public class OAuthBearerMiddlewareTests
         {
             var server = CreateServer(options =>
             {
+                options.AutomaticAuthentication = true;
```

Comment:
Given that the OAuth2 bearer middleware is now passive by default, setting this property to true is now mandatory. IMHO, it's not terribly intuitive and it should be automatic by default, just like it was in Katana.

https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNet.Authentication.OAuthBearer/OAuthBearerAuthenticationOptions.cs

http://katanaproject.codeplex.com/SourceControl/latest#src/Microsoft.Owin.Security/AuthenticationOptions.cs